### ISSUE TITLE
[Test Failure][Accessibility] String ‘Prerelease’ under package button is covered by its package description in PM UI

### ISSUE THREAD

**Issue by v-luzh** [NuGet/Home#4504]
**NuGet Version:** Dev 2265 VS15 Localized NuGet Client Vsix
**VS Version:** D15Rel 26203.00 (select all languages and workloads)
**OS Version:** Windows 10 RS1_Release 14393.821.170127-1801

**Repro Steps:**
1.  Select RUS through ‘Tools -> Options -> International Settings’.
2.  Create a new **C# .NET Core Unit Test Project** via VS.
3.  Right-click on the project and click ‘Manage NuGet Packages…’.
4.  Open **‘Installed’** tab.

**Expected Result:**
String ‘Prerelease’ under installed packages button is not covered by its package description.

**Actual Result:**
String ‘Prerelease’ under installed packages button is covered by its package description as below screenshot.
![prerelease](https://cloud.githubusercontent.com/assets/20632625/22618330/9a4d5404-eb14-11e6-9eb0-5d15d54e6a2c.png)

**Note:**
1. It repro on RUS,CSY,PTB,DEU,ITA,PLK,ESN,FRA.
2. It don’t repro on TRK,KOR,CHS,CHT,JPN.

**Comment by mishra14** [comment]
Seems like an accessibility bug to me.

**Comment by v-luzh** [comment]
This also repro on OS: Win10 RTM RUS+VS 2015 U3 RUS.
![4504reproondev14](https://cloud.githubusercontent.com/assets/20632625/23011220/ba5227ee-f45a-11e6-964b-03dd14a67f1c.PNG)

**Comment by v-luzh** [comment]
Fixed on D15.8 27719.00 + NuGet Client Dev 4.8.0.5215.
